frontend/src/pages/Category/index.jsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import { useParams } from 'react-router-dom';
+
```

Comment:
?

Reply:
if you are referring to the blank line: it separates our dependencies from our own components